pytorch_lightning/trainer/trainer.py

```diff
@@ -451,6 +451,7 @@ def __init__(
             amp_level,
             plugins,
         )
+        self._accelerator_connector.training_type_plugin
```

Comment:
note to self: need to find a way to avoid this. at the very least, this should be done within the connector

Reply:
Yeah agree, at least we should wet it in accelerator_connector

Reply:
actually, since it is no longer a property I think this line is now redundant. It does not do anything and let's try to remove it.

Reply:
Oh! good catch, it was added for debugging. Now accelerator_connector will initialized the ttp